calc/vxlan_resolver.go

```diff
@@ -483,6 +537,14 @@ func (c *VXLANResolver) determineGatewayForRoute(r vxlanRoute) string {
 // vtepMACForHost calculates a deterministic MAC address based on the provided host.
 // The returned address matches the address assigned to the VXLAN device on that node.
 func (c *VXLANResolver) vtepMACForHost(nodename string) string {
+	mac, ok := c.nodeNameToVXLANMac[nodename]
+
+	if ok {
```

Comment:
Don't need the extra "ok" check here.  If value isn't present, map will return `""`

Reply:
Acknowledged and changed.